namespaces/cloud-platform-live-0.k8s.integration.dsd.io/davids-dummy-dev/resources/ecr.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "davids_dummy_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.0"
```

Comment:
Did you want to change this to `ref=2.1`?

Reply:
Thanks. I've changed it.